docs/zh_cn/04-supported-codebases/mmcls.md

```diff
@@ -0,0 +1,19 @@
+# mmcls 模型支持列表
+
+[MMClassification](https://github.com/open-mmlab/mmclassification) 是基于 Python 的的图像分类工具，属于 [OpenMMLab](https://openmmlab.com)。
+
+## 安装 mmcls
+
+请参考 \[install.md\]（https://github.com/open-mmlab/mmclassification/blob/master/docs/en/install.md） 进行安装。
+
+## 支持列表
+
+| Model             | ONNX Runtime | TensorRT | ncnn | PPLNN | OpenVINO |                                          Model config                                           |
```

Comment:
could we add `torchscript` backend as well?

Reply:
I just copy it from `docs/en` directory.